flags.html

```diff
@@ -34,7 +34,15 @@
     </div>
 </div>
 <div id="sweden" class="flagContainer">
```

Comment:
oof, that's a painful way to do it.  If you need to change the dimensions, then you have to move multiple pieces.

Reply:
try something like this:  use horizontal and vertical bars.  2 elements to position rather than 4!

transform: translate makes it easier to move it relative to the size of the bar itself.

https://jsbin.com/fezobiy/3/edit?html,css,output